tests/receivers/postgresql/testdata/client/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG POSTGRES_VERSION=13-alpine
```

Comment:
Rather than copying the image files from the SA tests I think you can reference them by their existing paths like is done here: https://github.com/signalfx/splunk-otel-collector/blob/main/tests/receivers/smartagent/collectd-postgresql/collectd_postgresql_test.go#L26

The same thing can be done with the pod file references. Ideally we'd have a broken out shared k8s cluster helper but since it's just the two tests atm it's probably too early for that.